### PR TITLE
add one more c # book

### PR DESCRIPTION
**Pull request by Debajyotighosh654**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-ar.md [hunk 1]
```diff
@@ -36,6 +36,7 @@
 ### C Sharp
 
 * [سبيلك المختصر لتعلم لغة c# - برمجة الواجهات](https://www.mobarmijoun.com/2014/04/c_19.html) - Khaled Al-Saadani (Arab Coders Academy)
+*( https://www.introprogramming.info/wp-content/uploads/2013/07/Books/CSharpEn/Fundamentals-of-Computer-Programming-with-CSharp-Nakov-eBook-v2013.pdf)-Svetlin Nakov
```
Comment (eshellman) [inline books/free-programming-books-ar.md:39]: check your markdown syntax- there's no title?? and missing an author?

File: books/free-programming-books-ar.md [hunk 2]
```diff
@@ -36,6 +36,7 @@
 ### C Sharp
 
 * [سبيلك المختصر لتعلم لغة c# - برمجة الواجهات](https://www.mobarmijoun.com/2014/04/c_19.html) - Khaled Al-Saadani (Arab Coders Academy)
+* Fundamentals-of-Computer-Programming-with-CSharp - ( https://www.introprogramming.info/wp-content/uploads/2013/07/Books/CSharpEn/Fundamentals-of-Computer-Programming-with-CSharp-Nakov-eBook-v2013.pdf) (pdf)- (author Svetlin Nakov)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:39]: I could help you with resources format :wink:. Use this snippet

```markdown
* [FUNDAMENTALS OF COMPUTER PROGRAMMING WITH C# (The Bulgarian C# Programming Book)](https://www.introprogramming.info/wp-content/uploads/2013/07/Books/CSharpEn/Fundamentals-of-Computer-Programming-with-CSharp-Nakov-eBook-v2013.pdf) - Svetlin Nakov, Veselin Kolev et al. (PDF)
```

:information_source: Main authoritative site https://introprogramming.info/english-intro-csharp-book/ which this PDF can be found in.


:warning: But the **most important**. this resource is written in English. **Move it right listing**: [`free-programming-books-langs.md#c-sharp`](https://github.com/Debajyotighosh654/free-programming-books/blob/patch-2/books/free-programming-books-langs.md#c-sharp)

**Review by davorpa — CHANGES_REQUESTED** [review]
:warning: This resource is written in English. 

**Move it right listing**: [`free-programming-books-langs.md#c-sharp`](https://github.com/Debajyotighosh654/free-programming-books/blob/patch-2/books/free-programming-books-langs.md#c-sharp)

**Comment by eshellman** [comment]
still wrong markdown syntax

**Comment by davorpa** [comment]
This was merged at #5712. Please, avoid to reopen in the future

**Pull request closed without merging**